app/services/preview_draft_edition_service/payload.rb

```diff
@@ -14,43 +14,42 @@ def initialize(edition, republish: false)
 
   def payload
     payload = {
-      "locale" => edition.locale,
-      "schema_name" => publishing_metadata.schema_name,
-      "document_type" => document_type.id,
-      "publishing_app" => PUBLISHING_APP,
-      "rendering_app" => publishing_metadata.rendering_app,
-      "update_type" => edition.update_type,
-      "details" => details,
-      "links" => links,
-      "access_limited" => access_limited,
-      "auth_bypass_ids" => auth_bypass_ids,
+      locale: edition.locale,
+      schema_name: publishing_metadata.schema_name,
+      document_type: document_type.id,
+      publishing_app: PUBLISHING_APP,
+      rendering_app: publishing_metadata.rendering_app,
+      update_type: edition.update_type,
+      details: details,
+      links: links,
+      access_limited: access_limited,
+      auth_bypass_ids: auth_bypass_ids,
     }
-    payload["change_note"] = edition.change_note if edition.major?
+    payload[:change_note] = edition.change_note if edition.major?
 
     document_type.contents.each do |field|
-      attributes = field.payload(edition)
-      payload.deep_merge!(attributes.deep_stringify_keys)
+      payload.deep_merge!(field.payload(edition))
     end
 
     if edition.backdated_to.present?
-      payload["first_published_at"] = edition.backdated_to
-      payload["public_updated_at"] = edition.backdated_to if edition.first?
+      payload[:first_published_at] = edition.backdated_to
+      payload[:public_updated_at] = edition.backdated_to if edition.first?
     end
 
     if republish
-      payload["update_type"] = "republish"
-      payload["bulk_publishing"] = true
+      payload[:update_type] = "republish"
+      payload[:bulk_publishing] = true
     end
 
-    payload
+    payload.deep_stringify_keys
```

Comment:
do you know if we actually need this? would have thought it gets converted to JSON at some point so key types aren't a concern.

I guess it might break the WhitehallImporter code that also uses this class (I think the class should also be moved but that's something for outside this scope)

Reply:
We don't, and your guess was correct. I had another go and it turned out to be simpler than when I tried the first time - I've added another commit.